index.html

```diff
@@ -78,6 +78,165 @@ <h1 class="heading">
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-5 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-tile">
+            <div class="box-6 project-tile">
+              <h1 class="tile-heading">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h1>
+              <ul class="tile-ul">
+                <li class="tile-1">Ruby on rails</li>
+                <li class="tile-2">CSS</li>
+                <li class="tile-3">Javascript</li>
+                <li class="tile-4">Html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- About Me -->
+      <section>
+        <div class="about-me">
+          <h1 class="heading">About Me</h1>
+          <p class="description">
+            I can help you build a product,feature or website look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, don't hesitate to contact me.
+          </p>
+          <button>Get My Resume</button>
+        </div>
+      </section>
+
+      <!-- Language,framework & skills -->
+      <section class="abilities-section">
+        <div class="abilities">
+          <img src="./images/language.png" alt="language logo" />
+          <h3 class="heading-three">Languages</h3>
+          <ul class="language">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+
+        <div class="abilities">
+          <img src="./images/Frameworks.png" alt="framework logo" />
+          <h3 class="heading-three">Frameworks</h3>
+          <ul class="framework">
+            <li>Bootstrap</li>
+            <li>Ruby On Rails</li>
+            <li>Rspec</li>
+            <li>Selenium</li>
+            <li>Capybara</li>
+          </ul>
+        </div>
+
+        <div class="abilities">
+          <img src="./images/Skills.png" alt="skills logo" />
+          <h2 class="heading-three">Skills</h2>
+          <ul class="skills">
+            <li>Codekit</li>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` tag, each of them in a `li` tag as they are list, kindly implement it